packages/fx-core/src/common/versionMetadata.ts

```diff
@@ -18,6 +18,8 @@ export const MetadataV3 = {
   teamsManifestFileName: "manifest.json",
   aadManifestFileName: "aad.manifest.json",
   v3UpgradeWikiLink: "https://aka.ms/teams-toolkit-5.0-upgrade",
+  secretFileComment:
```

Comment:
please also add comment for `.env.dev` and `.env.local`, which has different strings.

Reply:
done